<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolDialog</class>
 <widget class="QDialog" name="ToolDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>730</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="rootLayout" stretch="0,1,0,0">
   <item>
    <widget class="QStackedWidget" name="stackedLoadWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="singleLoad">
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="OtoFileLoadWidget" name="otoLoadWidget" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="mulptipleLoad">
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="OtoFileMultipleLoadWidget" name="otoMultipleLoadWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="optionGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Processing options</string>
     </property>
     <layout class="QVBoxLayout" name="optionLayout">
      <item>
       <widget class="QWidget" name="presetWidgetContainer" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedSaveWidget">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="singleSave">
      <layout class="QGridLayout" name="saveWidgetLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="OtoFileSaveWidget" name="otoSaveWidget" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="multipleSave">
      <layout class="QGridLayout" name="gridLayout_4">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="OtoFileMultipleSaveWidget" name="otoMultipleSaveWidget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="switchLoadModeButton">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Switch between &lt;span style=&quot; font-weight:600;&quot;&gt;single file&lt;/span&gt; and &lt;span style=&quot; font-weight:600;&quot;&gt;batch (multiple file) &lt;/span&gt;mode. Current open state will be cleared.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Switch open mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OtoFileLoadWidget</class>
   <extends>QWidget</extends>
   <header>utils/widgets/otofileloadwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OtoFileSaveWidget</class>
   <extends>QWidget</extends>
   <header>utils/widgets/otofilesavewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OtoFileMultipleLoadWidget</class>
   <extends>QWidget</extends>
   <header>utils/widgets/otofilemultipleloadwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>OtoFileMultipleSaveWidget</class>
   <extends>QWidget</extends>
   <header>utils/widgets/otofilemultiplesavewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ToolDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>697</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ToolDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>697</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
